### DISCUSSION TITLE
Link Prediction on Multiple graph

### DISCUSSION THREAD

**Discussion by ankitjsrnit** [pyg-team/pytorch_geometric#7977] [Q&A]
Hi, 
i have multiple graph dataset. Each of my graph represents the unique PDF document, where node is component of pdf doc like 'title', 'paragraph', 'header', 'images' etc. I am trying to predict the the link between two component if exist. e.g. "a paragraph can only be connected to one header". I already implemented graph sage using other library. Now, i wanted to use torch geometric to implement my network. can you please share some example notebook or code with similar kind of problem statement.

Thanks,

**Comment by rusty1s** [comment]
How did you previously implement this? In the end, this may boil down to an edge classification or link prediction task, but you would still need some post-processing of predictions to confirm that constraints are met.
